spec/models/day_post.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
```

Comment:
It looks like this file shouldn't exist? This one doesn't end with "_spec" and there's almost another identical file called `day_post_spec.rb` in the same directory.

Reply:
my most recent push has reflected this

Reply:
should this file not be deleted?